dct_backend/webscraper.py

```diff
@@ -0,0 +1,111 @@
+import requests
+from bs4 import BeautifulSoup
+import dct
+
+# DCT import
+DCT = dct.DCT()
```

Comment:
The server will do this, not the scraper

Reply:
How will the server do it? I still need to import the other method, right?

Reply:
No, your methods should just parse the soup and return the data. Let the server handle what it wants to do with it.

Reply:
How do I save it then? (line 48)

Reply:
Have your methods return the data and the server will do that (line 48)